ui/src/polyfills.ts

```diff
@@ -70,9 +70,14 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
  */
-import 'zone.js/dist/zone';  // Included with Angular CLI.
-
+import 'zone.js/dist/zone';
+import {environment} from "./environments/environment";  // Included with Angular CLI.
 
+if(!environment.production){
+  (Map.prototype as any).toJSON = function () {
```

Comment:
Why is this needed?

Reply:
It is only for development because of !environment.production.
It is for Redux dev tools chrome plugin that I can see map content.